### PR TITLE
Define larval dipping form 

### PR DESCRIPTION
**Pull request by vincent-karuri**
PR for #115.

Depends on https://github.com/OpenSRP/opensrp-client-native-form/pull/174.

### CODE REVIEW

File: opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+      "openmrs_entity_id": "163152AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "encounter_location": ""
+  },
+  "step1": {
+    "title": "Larval Dipping Details",
+    "display_back_button": "true",
+    "fields": [
+      {
+        "key": "task_business_status",
```
Comment (githengi) [inline opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json]: This should be mandatory

File: opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+        "openmrs_entity_id": "",
+        "hint": "Date larval dipping was done",
+        "max_date": "today",
+        "v_required": {
+          "value": true,
+          "err": "Please specify the date larval dipping was done"
+        }
+      },
+      {
+        "key": "dips",
```
Comment (githengi) [inline opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json]: This should be mandatory

File: opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+        "type": "repeating_group",
+        "reference_edit_text_hint":"# of dips",
+        "repeating_group_label": "dip",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "value": [
+          {
+            "key": "larvae_total",
```
Comment (githengi) [inline opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json]: This should be mandatory

**Comment by githengi** [comment]
Resolve conflicts

File: opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json (hunk 3 above)
Comment (vincent-karuri) [reply opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json]: Resolved.

File: opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json (hunk 1 above)
Comment (vincent-karuri) [reply opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json]: Resolved.

File: opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json (hunk 2 above)
Comment (vincent-karuri) [reply opensrp-reveal/src/main/assets/json.form/thailand_larval_dipping_form.json]: Resolved.

File: opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/ListTaskPresenter.java [hunk 4]
```diff
@@ -284,8 +286,8 @@ private void formatSprayCardDetails(SprayCardDetails sprayCardDetails) {
         CardDetailsUtil.formatCardDetails(sprayCardDetails);
     }
 
-    private void startForm(Feature feature, CardDetails cardDetails, String encounterType) {
-        String formName = RevealJsonFormUtils.getFormName(encounterType, null);
+    private void startForm(Feature feature, CardDetails cardDetails, String interventionType) {
```
Comment (githengi) [inline opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/ListTaskPresenter.java]: The name of the variable here is actually the event `encounterType`

**Pull request merged**